packages/react/src/components/IconButton/index.tsx

```diff
@@ -150,7 +150,7 @@ export interface IconButtonProps
   /**
    * Specify the size of the Button.
    */
-  size?: Extract<ButtonSize, 'sm' | 'md' | 'lg'>;
+  size?: Extract<ButtonSize, 'xs' | 'sm' | 'md' | 'lg'>;
```

Comment:
The accessibility task isn't marked as completed in https://github.com/carbon-design-system/carbon/pull/19435#issue-3077573646. Do `xs` `IconButton`s meet https://www.ibm.com/able/requirements/requirements/?version=v7_3#2_5_8?

Reply:
Yes, xs buttons are 24 x 24 pixels, so just meet accessibility requirements